modules/oe_theme_helper/src/Plugin/Field/FieldFormatter/FeaturedMediaFormatter.php

```diff
@@ -88,13 +86,19 @@ public static function create(ContainerInterface $container, array $configuratio
   public static function defaultSettings() {
     return [
       'image_style' => '',
+      'view_mode' => 'oe_theme_main_content',
     ] + parent::defaultSettings();
   }
 
   /**
    * {@inheritdoc}
    */
   public function settingsForm(array $form, FormStateInterface $form_state) {
+    foreach (\Drupal::service('entity_display.repository')->getViewModes('media') as $key => $value) {
```

Comment:
\Drupal::service('entity_display.repository') shouldn't be used. Use container injection instead.